src/pages/groupComparison/GroupComparisonMutationsTab.tsx

```diff
@@ -41,18 +41,20 @@ export default class GroupComparisonMutationsTab extends React.Component<
     @action.bound
     protected handleGeneChange(id: string) {
         this.props.urlWrapper.updateURL({
-            selectedGene: id,
+            selectedGene: id ? id.replace('comp-tab-', '') : id,
         });
     }
 
     @computed get tabs() {
-        return this.props.store.genesWithMaxFrequency.map(g => (
-            <MSKTab
-                key={g.hugoGeneSymbol}
-                id={g.hugoGeneSymbol}
-                linkText={g.hugoGeneSymbol}
-            />
-        ));
+        return this.props.store.genesSortedByMutationFrequency
+            .result!.slice(0, 10)
+            .map(g => (
+                <MSKTab
+                    key={`comp-tab-${g}`}
+                    id={`comp-tab-${g}`}
```

Comment:
ok, i misled you here.  key should be prefixed but not the id property. that will simplify the selected gene thing above.  sorry

Reply:
fixed